lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -998,6 +998,8 @@ SILInstruction *SILCombiner::visitCondBranchInst(CondBranchInst *CBI) {
       return nullptr;
 
     SILBasicBlock *Default = nullptr;
+    unsigned NumTrueBBCases = 0;
+    unsigned NumFalseBBCases = 0;
```

Comment:
nitpick: it's better to move those 2 lines down to their first use
(I was confusing these variables with True/FalseBBCases inside the if-block)